eslint-config-feedzai-react/rules/react.js

```diff
@@ -116,6 +116,15 @@ module.exports = {
 
         // Prevent missing parentheses around multiline JSX
         "react/jsx-wrap-multilines": "error",
+        "react/jsx-wrap-multilines": ["error", {
+            "declaration": "parens-new-line",
+            "assignment": "parens-new-line",
+            "return": "parens-new-line",
+            "arrow": "parens-new-line",
```

Comment:
Regarding this one, could we change this one to `parens`? 

Because some thins this is useful in unit tests code like:

```javascript
jest.mock("xpto", () => (<div>Mocked component</div>));
```